<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWidgetVerticalLayout</class>
 <widget class="QWidget" name="AcquisitionWidgetVerticalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>557</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acquisition widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="acq_gbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="osc_start_label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Oscillation start (°):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="osc_start_cbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Oscillation start:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="osc_start_ledit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="osc_range_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Range per frame (°):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="osc_range_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="num_images_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Number of images:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="num_images_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="osc_total_range_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Total range (°):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="osc_total_range_ledit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="first_image_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>First image:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="first_image_ledit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="detector_roi_mode_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Detector mode:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="detector_roi_mode_combo">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Select necesaary detector mode</string>
          </property>
          <item>
           <property name="text">
            <string>9M</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4M</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="exp_time_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Exposure time (s):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="exp_time_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="total_exp_time_label">
          <property name="text">
           <string>Total exp. time (s):</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="exp_time_total_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="kappa_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Kappa (°):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="kappa_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="kappa_phi_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Phi (°):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="kappa_phi_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="energy_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Energy (keV):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="energy_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QCheckBox" name="mad_cbox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Check to use energies from energy scan</string>
          </property>
          <property name="text">
           <string>MAD</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QComboBox" name="energies_combo">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Check to use energies from energy scan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="resolution_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Resolution (Å):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="resolution_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="detector_distance_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Detector distance (mm):</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="detector_distance_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="transmission_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Transmission (%):</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="transmission_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="flux_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Flux (ph/s):</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLineEdit" name="flux_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="shutterless_cbx">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Shutterless</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="dose_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Estimated dose (MGy):</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLineEdit" name="dose_ledit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="2">
         <widget class="QCheckBox" name="max_osc_range_cbx">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Full osc. range</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="set_max_osc_range_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Allowed range:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QPushButton" name="set_max_osc_range_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Set maximum allowed oscillation range</string>
          </property>
          <property name="text">
           <string>Full range</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
